src/DripsHub.sol

```diff
@@ -931,32 +870,8 @@ abstract contract DripsHub {
         amtDelta.nextCycle += int128(uint128(nextCycleSecs)) * amtPerSecDelta;
     }
 
-    function _userOrAccount(address user) internal pure returns (UserOrAccount memory) {
-        return UserOrAccount({isAccount: false, user: user, account: 0});
-    }
-
-    function _userOrAccount(address user, uint256 account)
-        internal
-        pure
-        returns (UserOrAccount memory)
-    {
-        return UserOrAccount({isAccount: true, user: user, account: account});
-    }
-
-    function calcUserId(UserOrAccount memory userOrAccount) public pure returns (uint256) {
-        if (userOrAccount.isAccount) {
-            return calcUserId(userOrAccount.user, userOrAccount.account);
-        } else {
-            return calcUserId(userOrAccount.user);
-        }
-    }
-
     function calcUserId(address user) public pure returns (uint256) {
-        return uint256(keccak256(abi.encode(user)));
-    }
-
-    function calcUserId(address user, uint256 account) public pure returns (uint256) {
-        return uint256(keccak256(abi.encode(user, account)));
+        return uint160(user);
```

Comment:
I see this is also part of the intermediate code.

Reply:
It is, it's dropped in https://github.com/radicle-dev/radicle-drips-hub/pull/115.